integration-tests/templates/owner-onboarding-server.yml.j2

```diff
@@ -10,8 +10,29 @@ owner_addresses_path: {{ config_dir }}/owner-addresses.yml
 report_to_rendezvous_endpoint_enabled: true
 bind: {{ bind }}
 service_info:
-  rhsm_organization_id: 6454383
+  rhsm_organization_id: 123456
   rhsm_activation_key: SDOkey
   rhsm_run_insights: true
   sshkey_user: {{ user }}
   sshkey_key: "testkey"
+  files:
```

Comment:
https://github.com/coreos/ignition/pull/1285 ugh

Reply:
Hmm, okay. So you want me to rename this?

Reply:
oh no, just wondering when/if we can use something like that to avoid reimplementing file handling

Reply:
So, the thought here is that right now we don't have any other useful serviceinfo modules.
This one together with the process one could be used to kickstart that, by for example putting in a file with an ignition or ansible file, and then have the command module call ignition/ansible.
i.e. this is just a basic one in order to allow for those types of use cases.